tests/foreman/ui/test_contentview.py

```diff
@@ -2640,55 +2545,6 @@ def test_positive_search_composite(session):
         }
 
 
-@pytest.mark.tier3
-@pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_publish_with_force_puppet_env(session, module_org):
-    """Check that puppet environment will be created automatically once
-    content view that contains puppet module is published, no matter
-    whether 'Force Puppet' option is enabled or disabled for that content
-    view
-    Check that puppet environment will be created automatically once content
-    view without puppet module is published, only if 'Force Puppet' option is
-    enabled
-
-    :id: af553367-e621-41e8-86cb-091ba7ba6c0a
-
-    :customerscenario: true
-
-    :expectedresults: puppet environment is created only in expected cases
-
-    :BZ: 1437110
-
-    :CaseLevel: Integration
-
-    :CaseImportance: High
-    """
-    puppet_module = 'httpd'
-    create_sync_custom_repo(
-        module_org.id, repo_url=settings.repos.puppet_0.url, repo_type=REPO_TYPE['puppet']
-    )
-    with session:
-        for add_puppet in [True, False]:
-            for force_value in [True, False]:
-                cv_name = gen_string('alpha')
-                session.contentview.create({'name': cv_name})
-                session.contentview.update(cv_name, {'details.force_puppet': force_value})
-                if add_puppet:
-                    session.contentview.add_puppet_module(cv_name, puppet_module)
```

Comment:
I think this still can be implemented without `add_puppet_module` 
You can use [workaround fixture](https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L535) to get puppet repo.
We will still test at least force value. I don't have knowledge how that option actually works, so decision is up to you.